debezium-core/src/main/java/io/debezium/config/Configuration.java

```diff
@@ -2125,6 +2125,15 @@ default String getFallbackStringProperty(Field newProperty, Field oldProperty) {
      * @return the evaluated value
      */
     static String getFallbackStringProperty(Configuration config, Field newProperty, Field oldProperty) {
+        if (null != config.getString(oldProperty.name()) && null != config.getString(newProperty.name())) { // both are set
+            CONFIGURATION_LOGGER.warn("Provided configuration has deprecated property \"" + oldProperty.name()
+                    + "\" and new property \"" + newProperty.name() + "\" set. Using value from \"" + newProperty.name() + "\"!");
+        }
+        return config.getString(newProperty,
+                () -> config.getString(oldProperty));
+    }
+
+    static String logFallbackStringProperty(Configuration config, Field newProperty, Field oldProperty) {
```

Comment:
Having removed the static method above, I'd suggest to rename this one into `getFallbackStringPropertyWithWarning()`, also adding a JavaDoc comment to the method (no need for param/return value tags, as those are evident -- we overdid this a bit in the past --).